.circleci/main.yml

```
@@ -227,7 +228,7 @@ jobs:
           command: |
             npm init -y
             npm install ipfs@^0.59.1
-            npm install ipfs-interop@^7.0.3
+            npm install ipfs/interop#feat/go-libp2p-0.16-and-sharding-config-option
```

@aschmahmann I propose we merge https://github.com/ipfs/interop/pull/400  first, and then flip this line `master` before merging to master here:

```suggestion
            npm install ipfs/interop#master
```

This way, we don't need to change this line in go-ipfs' `master` until a release version of ipfs/interop is tagged after all the other stuff lands.

---

Merged: https://github.com/ipfs/interop/pull/400